src/content/self-hosted/administration/configuration.mdx

```diff
@@ -517,6 +517,24 @@ The role Okteto assigns to every user. If empty, Okteto will create a default ro
 clusterRole: "role name"
 ```
 
+### user extraRoleBindings
+
+Extra `RoleBindings` to be created for every user's service account. This is useful if you want to grant access to other namespace resources in the cluster.  Disabled by default.
+
+The `RoleBindings` will reference `ClusterRoles` and will be scoped to the namespace specified in the configuration. Here's an example:
+
+```yaml
+user: 
```

Comment:
@pchico83 is it `user`or `users`? (just want to make sure it's all the same :P)

Reply:
As it is now implemented the key is `user`, but I can do the work to replace it with `users`

Reply:
If `user` is the right one, leave it that way 👍